client/src/modules/invoices/registry/modalCreditNote.html

```diff
@@ -31,14 +31,27 @@
         <tbody>
           <tr ng-repeat="item in ModalCtrl.patientInvoiceItems track by item.uuid">
             <td>{{item.text}}</td>
-            <td>{{item.quantity}}</td>
-            <td>{{item.transaction_price | currency:ModalCtrl.patientInvoice.currency_id }}</td>
-            <td>{{item.quantity * item.transaction_price | currency}}</td>
+            <td class="text-right">{{item.quantity}}</td>
+            <td class="text-right">{{item.transaction_price | currency:ModalCtrl.patientInvoice.currency_id }}</td>
+            <td class="text-right">{{item.quantity * item.transaction_price | currency}}</td>
           </tr>
+          <tr>
+            <td colspan="4"></td>
+          </tr>
+          <tr ng-show="ModalCtrl.billingAmount>0">
+              <td translate>FORM.LABELS.BILLING_SERVICES</td>
+              <td class="text-right" colspan="3">{{ ModalCtrl.currencySymbol + ModalCtrl.billingAmount }}</td>
+          </tr>
+          <tr ng-show="ModalCtrl.subsidyAmount>0">
+            <td translate>FORM.LABELS.SUBSIDIES</td>
+            <td class="text-right" colspan="3">-{{ ModalCtrl.currencySymbol + ModalCtrl.subsidyAmount }}</td>
```

Comment:
Could you add a space between the "-" and the "{{"?  It would make it easier to read.